src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/index.jsx

```diff
@@ -160,22 +167,13 @@ const SettingsDescription = ({ action, addedOwner, newThreshold, removedOwner }:
   )
 }
 
-const CustomDescription = ({ amount = 0, classes, data, recipient }: CustomDescProps) => {
-  const [showTxData, setShowTxData] = useState(false)
-  const recipientName = useSelector((state) => getNameFromAddressBook(state, recipient))
+const TxData = (props: TxDataProps) => {
+  const { classes, data, setShowTxData, showTxData } = props
+  const showLongData = data.length > 20
```

Comment:
just a nitpick, let's name this "showExpandBtn"

Reply:
done